README.md

```diff
@@ -76,6 +77,29 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+<a name="why_globalize"></a>
+### Why Globalize?
+
+Globalize has initially been created to fullfil jQuery UI (datepicker and
+spinner widgets). It provides number formatting and parsing, date and time
+formatting and parsing, currency formatting, message formatting (ICU message
+format pattern), and plural support.
+
+Our design goals are (a) to leverage the official CLDR JSON data, (b) allow
+users to load as much or as little data as they need, (c) avoid duplicating data
+if using multiple i18n libraries that leverage CLDR, and (d) run in browsers or
+node.js.
+
+Some of the strengths of using Globalize are:
+
+- Based on CLDR data (largest and most extensive i18n repository).
+- No bugs due to outdated i18n content (it has no hard coded i18n data).
+- Consistency. Same results are obtained across browsers.
+- Allow different locale instances running simultaneously.
+
+For an overview of JavaScript globalization libraries, check out [this
```

Comment:
Rewriting myself: "For alternative libraries and more, check out this [JavaScript globalization overview]."